app/assets/javascripts/dropdown.js

```diff
@@ -29,3 +28,55 @@ $("#country-select").on("change", function() {
     }
   });
 });
+
+
+function initialize() {
+  var locations = $('#map-canvas').data("locations");
+  //debugger;
+  var mapProp = {
+    center:{lat: 40, lng: 290},
+    zoom:1,
+    backgroundColor: '#0e1525',
+    mapTypeId:google.maps.MapTypeId.ROADMAP
+  };
+  var map=new google.maps.Map(document.getElementById("map-canvas"),mapProp);
+  for (var i = 0; i < locations.length; i++) {
+    new google.maps.Marker({
+      position:{lat: locations[i]['lat'], lng: locations[i]['lon']},
```

Comment:
Mixed double and single quotes.<br>['lat'] is better written in dot notation.<br>['lon'] is better written in dot notation.